package.json

```diff
@@ -82,20 +82,27 @@
     "doc": "esdoc -c esdoc.json",
     "dev": "gulp --deploy-env dev dev",
     "minify": "gulp --deploy-env production minify",
-    "postinstall": "lerna bootstrap",
-    "prepare": "gulp --deploy-env production prepublish",
+    "postinstall": "npm run lerna-bootstrap && npm run npm-install-lock-only",
+    "prepare": "npm run build-production",
+    "lerna-bootstrap": "npm run lerna-bootstrap-core && npm run lerna-bootstrap-others",
+    "lerna-bootstrap-core": "lerna --scope skygear-core bootstrap",
+    "lerna-bootstrap-others": "lerna --ignore skygear-core bootstrap",
+    "build-production": "gulp --deploy-env production prepublish",
     "test": "gulp --deploy-env dev",
     "lerna": "lerna",
     "example": "gulp --deploy-env dev example",
     "npm-install": "lerna exec -- npm install",
+    "npm-install-lock-only": "lerna exec -- npm install --package-lock-only",
     "audit": "lerna exec -- npm audit"
   },
   "bin": {
     "skygear-node": "./bin.js",
     "skygear-asset": "./asset.js"
   },
   "license": "Apache-2.0",
-  "dependencies": {},
+  "dependencies": {
+    "util-deprecate": "^1.0.2"
```

Comment:
`util-deprecate` should goes into `skygear-core` (or whichever `skygear-*` package that uses `util-deprecate`)